Source/SwiftLintFramework/Helpers/RegexHelpers.swift

```diff
@@ -8,7 +8,7 @@
 
 struct RegexHelpers {
     // A single variable
-    static let varName = "[a-zA-Z_][a-zA-Z0-9_]+"
+    static let varName = "[a-zA-Z_\.][a-zA-Z0-9_\.]+"
```

Comment:
It also doesn't work with simple operations `NSMakeRange(length + 2, 0)`.

Is this something I should add a new test file for or where would tests belong? I just started looking at this code so it might take a few days to get up to speed but I'll add some regex/autocorrect tests if that makes sense?